src/shared/components/comment/comment-node.tsx

```diff
@@ -315,8 +315,12 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
                 <Icon icon="shield" inline classes="text-danger me-2" />
               )}
               {this.isPostCreator && (
-                <div className="badge text-bg-light d-none d-sm-inline me-2">
-                  {I18NextService.i18n.t("creator")}
+                <div
```

Comment:
This is probably more like a `<span>`. Bootstrap says `<span>` is ok: https://getbootstrap.com/docs/5.3/components/badge/#examples

Reply:
Good call.